src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -103,7 +103,7 @@ If somebody runs an Ethereum node with a public API in your community, you can p
 
 On the other hand, if you run a client, you can share it with your friends who might need it.
 
-## Clients {#clients}
+## Execution clients {#execution-clients}
 
 The Ethereum community maintains multiple open-source clients, developed by different teams using different programming languages. This makes the network stronger and more diverse. The ideal goal is to achieve diversity without any client dominating to reduce any single points of failure.
```

Comment:
```suggestion
The Ethereum community maintains multiple open-source execution clients (previously known as "Eth1 clients", or just "Ethereum clients"), developed by different teams using different programming languages. This makes the network stronger and more diverse. The ideal goal is to achieve diversity without any client dominating to reduce any single points of failure.
```

Reply:
Not sure this is the best language but I think we should call out the terminology update.